src/main/java/calculator/StringCalculator.java

```diff
@@ -0,0 +1,64 @@
+package calculator;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class StringCalculator {
+
+    public static final int ZERO = 0;
+    public static final String DEFAULT_DELIMITER = ",|:";
+    public static final String CUSTOM_DELIMITER_FIND_REGEX = "//(.*)\n(.*)";
+
+    public static int splitAndSum(String text) {
+        int sum = ZERO;
+        if (isBlank(text)) {
+            return sum;
+        }
+
+        sum = sum(getNumbers(split(text)));
+        return sum;
```

Comment:
```suggestion
    public static int splitAndSum(String text) {
        if (isBlank(text)) {
            return ZERO;
        }

        return sum(getNumbers(split(text)));
```
- 지역변수를 둠으로써 추가로 표현할 메시지나 이점이 따로 없어보어서요. 제거해도 되지 않을까요